src/components/BoxRegistrationModal.tsx

```diff
@@ -29,19 +35,29 @@ const BoxRegistrationModal: FC<BoxRegistrationModalProps> = (props: BoxRegistrat
           <Formik
             initialValues={{boxId: '', startDate: new Date()}}
             onSubmit={(values, {setSubmitting}) => {
-              const box = new Box(values.boxId, values.startDate);
+
               setSubmitting(true);
               setTimeout(() => {
-                void updateBoxForTitle(props.titleId, box)
-                  .then(res => {
-                    if (res.ok) {
-                      props.updateBoxInfo(box);
-                      props.closeModal();
+                void postNewBoxForTitle(props.titleId, values.boxId, values.startDate)
+                  .then(createdBox => {
+                    props.updateBoxInfo(createdBox);
+                    props.closeModal();
+
+                  })
+                  .catch(async (e: Error) => {
+                    if (e instanceof AlreadyExistsError) {
+                      await getBoxById(values.boxId).then((b: box) => {
+                        setExistingBox(b);
+                        if (b.title_id === +props.titleId) {
+                          setShowInfo({showModal: true, sameTitle: true});
+                        } else {
+                          setShowInfo({showModal: true, sameTitle: false});
```

Comment:
Har du prøvd å hente tittelnavn når det ikke er samme tittel? Ser for meg at det kan være kjekt for dem å se det uten å måtte åpne tittelen

Reply:
Kan legge det til

Reply:
Der var den oppdatert @MariusLevang